teams/vsa/design/getting-started-with-va.gov-forms.md

```diff
@@ -164,8 +167,15 @@ This document will not attempt to rewrite those documents, however, we will prov
    * Hierarchy/labelling
 
 ### Developers
-* Make sure you read up on and understand form builder [https://department-of-veterans-affairs.github.io/veteran-facing-services-tools/forms](https://department-of-veterans-affairs.github.io/veteran-facing-services-tools/forms)/ and/or [watch the quick demo Chris V. gave to the design team](https://zoom.us/rec/share/x5FpP7XC1DJOYZ3d933GUagdMInqeaa82ilM-KcLzUxo_-q1CWuJcUJVwzaPXTp_) in Feb. 2020 (pw: vsadesign)
-   * Fill out a few forms on staging.va.gov to get a feel for the capabilities of the forms system
+* Make sure you read up on and understand form builder:
+   *  Fill out a few forms on staging.va.gov to get a feel for the capabilities of the forms system.
+   *  [Watch the zoom video of a quick demo Chris Valarida gave to the design team](https://zoom.us/rec/share/x5FpP7XC1DJOYZ3d933GUagdMInqeaa82ilM-KcLzUxo_-q1CWuJcUJVwzaPXTp_) in Feb. 2020 (pw: vsadesign)
```

Comment:
Perhaps rather than pointing to zoom (which we do not control), we should download the .mp4 somewhere, make a link to that instead. That way we reduce the risk of losing it, plus we can get rid of the password and control migration of the video to somewhere else later.

Reply:
I agree, I think adding it to Git LFS makes sense since this will live in a Github doc

Reply:
Great, I added .mp4 to the repo's .gitattributes just now. I saw that @tlei123 has earlier added .m4v without much fanfare . You'll still have to to a -f with git add (force) because I left .mp4 alone in the .gitignore just to be safe.